Lesson_10/task2/task2.py

```diff
@@ -0,0 +1,252 @@
+"""
+Phonebook
+"""
+import json
+import os
+import argparse
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument("file_name", help="Name of Phonebook")
+args = parser.parse_args()
+
+current_dir = os.path.dirname(os.path.realpath(__file__))
+file_name = os.path.join(current_dir, args.file_name + ".json")
+temp_file = os.path.join(current_dir, "src", "temp.json")
+
+# Initialization for empty phonebook file
+if not os.path.exists(file_name):
+    input_code = input(
+        f"File with name '{args.file_name}' not found!\n"
+        "Do you want to create new file? (y/n)"
+    )
+    if not input_code.lower() == "y":
```

Comment:
User can potentially mistype something instead of "y", so I would suggest to check this case as well